infra/aws/terraform/audit-account/inspector.tf

```diff
@@ -0,0 +1,26 @@
+/*
+Copyright 2023 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+resource "aws_inspector2_delegated_admin_account" "default" {
+  account_id = local.audit-account-id
+}
+
+resource "aws_inspector2_organization_configuration" "default" {
+  auto_enable {
+    ec2 = true
+    ecr = false
```

Comment:
I'd expect the opposite. Do we not want to scan container images?

Reply:
Short term, we don't intent to build and push images to ECR. We can enable this where this is a need to host critical container images on ECR.